posts/kute-zelazo-vs-aluminium copy.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Jak działają bramy przesuwne?"
+date: "April 8, 2021"
+excerpt: "Bramy przesuwne stają się coraz bardziej popularne, zwłaszcza w miejscach, gdzie jest mało miejsca na swobodne funkcjonowanie bram skrzydłowych i gdzie wymagany jest wyższy poziom bezpieczeństwa. To naprawdę wygodna i świetna opcja"
+---
```

Comment:
Is it possible to have image for post? xD

Reply:
good idea to add this but I would leave it for a next pull request